docs/resources/Application.md

```diff
@@ -4,32 +4,35 @@
-| custom_install_url?                | string                                                                     | the application's default custom authorization link, if enabled                                                                                                             |
-| role_connections_verification_url? | string                                                                     | the application's role connection verification entry point, which when configured will render the app as a verification method in the guild role verification configuration |
+| Field                              | Type                                                                       | Description                                                                                                           |
+| ---------------------------------- | -------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| id                                 | snowflake                                                                  | ID of the app                                                                                                         |
+| name                               | string                                                                     | Name of the app                                                                                                       |
+| icon                               | ?string                                                                    | [Icon hash](#DOCS_REFERENCE/image-formatting) of the app                                                              |
+| description                        | string                                                                     | Description of the app                                                                                                |
+| rpc_origins?                       | array of strings                                                           | List of RPC origin URLs, if RPC is enabled                                                                            |
+| bot_public                         | boolean                                                                    | when `false`, only the app owner can add the app to guilds                                                            |
+| bot_require_code_grant             | boolean                                                                    | when `true`, the app's bot will only join upon completion of the full OAuth2 code grant flow                          |
+| bot?                               | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | Partial user object for the bot user associated with the app                                                          |
```

Comment:
Is there any chance the payload of this could be reworked to match what is returned from `/users/@me`?

Reply:
would suggest opening an issue about this for sure (if there hasn't been one opened already).